CHANGELOG.md

```diff
@@ -1,4 +1,50 @@
-v2.4.0 - UNRELEASED
+v 2.5.0 UNRELEASED
+======
+* Added support for [SpEL](https://docs.spring.io/spring/docs/5.2.7.RELEASE/spring-framework-reference/core.html#expressions) and [property placeholders](https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/context/support/PropertySourcesPlaceholderConfigurer.html) in `@Spec` attributes: `constVal`, `defaultVal`. 
+  To enable SpEL support:
+  * Configure `SpecificationArgumentResolver` by passing `AbstractApplicationContext` in constructor
```

Comment:
does it have to be "AbstractAppliactionContext"? Or maybe "ApplicationContext" is enough?

Reply:
it has to be `AbstractAppliactionContext`